echeck.c

```diff
@@ -2328,16 +2330,27 @@ char *getbuf(void)
             && isspace(*bp));
         }
       } else {
-        cont = false;
-        if (*bp == '"') {
-          quote = (bool) ! quote;
-          eatwhite = true;
+        if ((!quote || quote == *bp) && (*bp == DOUBLE_QUOTE || *bp == SINGLE_QUOTE)) {
```

Comment:
Diese Änderung musste ich rückgängig machen (commit 4cba246ab17b8b0bf6300c8ffc5777bf806d0158), weil sie nicht mit Kommentaren klar kam, die den Namen einer Einheit wie Patrick O'Connor klar kommt (die also ein einzelnes Anführungszeichen in einem Kommentar haben).